packages/spacecat-shared-data-access/src/service/import-job/accessPatterns.js

```diff
@@ -56,12 +61,12 @@ export const getImportJobByID = async (dynamoClient, config, log, id) => {
 };
 
 /**
- * Get Import jobs by status
+ * Get Import jobs by status.
  * @param {DynamoClient} dynamoClient
  * @param {Object} config
  * @param {Logger} log
  * @param {string} status
- * @returns {Promise<ImportJobDto[]>}
+ * @return {Array<ImportJob>} - The ImportJobs with the given status.
```

Comment:
Does the removal of `Promise` for these async functions affect anything downstream?

Reply:
I think that it was wrong for me to remove it.